addons/website/views/snippets/s_video.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_video" name="Video">
+    <div class="media_iframe_video o_snippet_drop_in_only" data-oe-expression="//www.youtube.com/embed/G8b4UZIcTfg?rel=0&amp;autoplay=0">
```

Comment:
Instead of a class should probably be in the t-snippet definition. But leaving like that for now. We'll refactore snippet definition at some point anyway.